src/styles/blocks/header.scss

```
@@ -26,6 +26,11 @@
     color: #060b35;
     list-style: none;
     line-height: 60px;
+    transition: 300ms;
```

```suggestion
    transition: color 300ms;
```
It is better to add color property here so we can know what the transition is for .